src/turtle/TurtleSoup.java

```diff
@@ -105,13 +108,14 @@ public static void drawRegularPolygon(Turtle turtle, int sides, int sideLength)
      */
     public static double calculateHeadingToPoint(double currentHeading, int currentX, int currentY,
                                                  int targetX, int targetY) {
-        int dX = targetX - currentX;
-        int dY = targetY - currentY;
-        double theta = 90 - Math.atan2(dY, dX) * 180 / Math.PI -currentHeading;
-        if (theta < 0 ){
-            return theta +360 ;
-        }else{
-            return theta ;
+        final int dX = targetX - currentX;
+        final int dY = targetY - currentY;
+        final double theta = 90 - Math.atan2(dY, dX) * 180 / Math.PI - currentHeading;
```

Comment:
You might want to explain the role of "90-" for transforming the angle from the domain of "Math.atan" to the domain of the spec.